Ginger/GingerCoreNET/Application Models/Learn/APIModels/ApiActionConversionUtils.cs

```diff
@@ -320,68 +441,39 @@ private void SetApplicationAPIModel(ref ApplicationAPIModel aPIModel, Act act, b
                     }
                 }
 
+                if (parameterizeRequestBody)
+                {
+                    ParameterizeApiModellBody(aPIModel, actionToConvert, parameterizeRequestBody);
+                }
+
                 if (pullValidations)
                 {
-                    foreach (var item in act.ReturnValues)
+                    foreach (var exitingRV in actionToConvert.ReturnValues)
                     {
-                        if ((!string.IsNullOrEmpty(item.Expected) &&
-                            (item.StoreTo == ActReturnValue.eStoreTo.None ||
-                             item.StoreTo == ActReturnValue.eStoreTo.ApplicationModelParameter)))
+                        ActReturnValue actR = new ActReturnValue();
+                                              
+                        if (string.IsNullOrEmpty(exitingRV.Expected) == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4513408)